boards/arm/mimxrt685_evk/Kconfig.board

```diff
@@ -4,5 +4,6 @@
 config BOARD_MIMXRT685_EVK
 	bool "NXP MIMXRT685-EVK"
 	depends on SOC_SERIES_IMX_RT6XX
+	select CODE_DATA_RELOCATION_SRAM
```

Comment:
This should be done in the `soc` as the file that is relocated into RAM is located in the `soc` folder

Reply:
@mmahadevan108 
I have moved this selection into the soc/Kconfig.series, hopefully this is a better spot.